docs/installation/autoware/source-installation.md

```diff
@@ -2,7 +2,17 @@
 
 ## Prerequisites
 
-- [Ubuntu 20.04](https://releases.ubuntu.com/20.04/)
+- OS
+
+  - [Ubuntu 20.04](https://releases.ubuntu.com/20.04/) (**active development**)
+  - [Ubuntu 22.04](https://releases.ubuntu.com/22.04/) (**will be supported in 2022**)
+
+- ROS
+
+  - ROS 2 Galactic (**active development**)
+  - ROS 2 Humble (**will be supported in 2022**)  
+    Refer to [REP-2000](https://www.ros.org/reps/rep-2000.html) for the system dependencies.
```

Comment:
I guess it might be better to write everything in the `Target platforms` section and link there, in order to aggregate information.
@Sharrrrk @LalithVipulananthan What do you think about it?

```suggestion
- See the [Target platforms](../index.md#target-platforms)
```

Reply:
@kenji-miyake but for docker installation, Ubuntu and ROS 2 are not required, so I only listed the hardware requirements in **Target platforms** chapter.